PocketKit/Sources/PocketKit/Tags/TagsFilterView.swift

```diff
@@ -34,7 +34,7 @@ struct TagsFilterView: View {
                     TagsCell(tag: .notTagged, tagAction: tagAction)
                         .disabled(isEditing)
                     TagsSectionView(
-                        showRecentTags: false,
+                        showRecentTags: viewModel.showRecentTags,
```

Comment:
@cyndichin I had to change some code for `test_archiveView_usingRecentTagFilter_withPremiumUser_showFilteredItems` to pass. is this intended?

Reply:
Ah the original code should be `showRecentTags: !isEditing && !viewModel.recentTags.isEmpty,`